src/strategy/util.ts

```diff
@@ -1,5 +1,23 @@
 import * as murmurHash3 from 'murmurhash3js';
 
-export default function normalizedValue(id: string, groupId: string, normalizer = 100): number {
-  return (murmurHash3.x86.hash32(`${groupId}:${id}`) % normalizer) + 1;
+function normalizedValue(id: string,
```

Comment:
making this private since I want other code to use strategy or variant specific functions